sources/osgWrappers/serializers/osgAnimation.js

```diff
@@ -238,5 +264,112 @@ define( [
         return st;
     };
 
+    osgAnimationWrapper.Bone = function ( input, bone ) {
+        var jsonObj = input.getJSON();
+        var check = function ( o ) {
+            if ( o.InvBindMatrixInSkeletonSpace && o.BoneInSkeletonSpace ) {
+                return true;
+            }
+            return false;
+        };
+        if ( !check( jsonObj ) ) {
+            return undefined;
+        }
+
+        var promise = osgWrapper.MatrixTransform( input, bone );
+
+        if ( jsonObj.InvBindMatrixInSkeletonSpace !== undefined ) {
+            bone.setInvBindMatrixInSkeletonSpace( jsonObj.InvBindMatrixInSkeletonSpace );
+        }
+        if ( jsonObj.BoneInSkeletonSpace !== undefined ) {
+            bone.setMatrixInSkeletonSpace( jsonObj.BoneInSkeletonSpace );
+        }
+        return promise;
+    };
+
+    osgAnimationWrapper.Skeleton = function ( input, skl ) {
+        var jsonObj = input.getJSON();
+        var check = function ( o ) {
+            if ( o.Matrix ) {
+                return true;
+            }
+            return false;
+        };
+        if ( !check( jsonObj ) ) {
+            return;
+        }
+
+        return osgWrapper.MatrixTransform( input, skl );
+    };
+
+    osgAnimationWrapper.FloatCubicBezierChannel = function ( input, channel ) {
+        var jsonObj = input.getJSON();
+        var check = function ( o ) {
+            if ( o.KeyFrames && o.TargetName && o.Name ) {
+                var KeyFrames = o.KeyFrames;
+                if ( KeyFrames.Time && KeyFrames.Position && KeyFrames.ControlPointOut && KeyFrames.ControlPointIn ) {
+                    return true;
+                }
+            }
+            return false;
+        };
+        if ( !check( jsonObj ) ) {
+            return;
+        }
+
+        //doit
+        if ( !osgWrapper.Object( input, channel ) ) {
+            return;
+        }
+
+        channel.setTargetName( jsonObj.TargetName );
+
+        //Keys building
+        var keys = channel.getSampler().getKeyframes();
+
+        var bezierChannel = {};
+        var arraysPromise = [];
+
+        var createChannelAttribute = function ( name, jsonAttribute ) {
+            var defer = Q.defer();
+            arraysPromise.push( defer.promise );
+
+            var promise = input.setJSON( jsonAttribute ).readBufferArray();
+            Q.when( promise ).then( function ( buffer ) {
+                if ( buffer !== undefined ) {
+                    bezierChannel[ name ] = buffer._elements;
+                }
+                defer.resolve( buffer );
+            } );
+        };
+
+        for ( var k in jsonObj.KeyFrames ) {
```

Comment:
the way to iterate on keys of an object is:
var keys = Objects.keys(jsonObj.KeyFrames);
for ( var i = 0; i < keys.lenght; i++ ) {
  var key = keys[i];
  var value = jsonObj.KeyFrames[key];
  // do something with value
}